runner.go

```diff
@@ -375,6 +401,33 @@ func (r *runner) makeRunParams(c cliContext, localTests []*rainforest.RFTest) (r
 	}, nil
 }
 
+func (r *runner) makeRerunParams(c cliContext) (rainforest.RunParams, error) {
+	var err error
+
+	var runID int
+	runIDString := c.Args().First()
+	if runIDString == "" {
+		runIDString = os.Getenv("RAINFOREST_RUN_ID")
+	}
+	if runIDString == "" {
+		return rainforest.RunParams{}, errors.New("Missing run ID")
+	}
+	runID, err = strconv.Atoi(runIDString)
+	if err != nil {
+		return rainforest.RunParams{}, errors.New("Invalid run ID specified")
+	}
+
+	var conflict string
+	if conflict = c.String("conflict"); conflict != "" && conflict != "abort" && conflict != "abort-all" {
```

Comment:
This validation logic is duplicated elsewhere, it could be moved to an `Action` block in the CLI parser?:
https://github.com/rainforestapp/rainforest-cli/blob/master/rainforest-cli.go#L193

Reply:
`Action` blocks aren't for flags. There's no first-class support for enums yet, but there's a workaround shown in the [open issue](https://github.com/urfave/cli/issues/602).

Reply:
Are you planning to use this workaround? I guess the downside of it is the custom `EnumValue` struct def we'd have to add/maintain. IMO a possible approach could be to extract a simple `validateConflict(c cliContext) (string, error)` func to avoid the duplicate check

Reply:
> IMO a possible approach could be to extract a simple validateConflict(c cliContext) (string, error) func to avoid the duplicate check

Planning on doing that and staying with the existing `StringFlag` for now.